components/cf-app-push/frontend/src/view/deploy-app-workflow/deploy-app.service.js

```diff
@@ -183,6 +215,45 @@
           });
       }
     };
+
+    /*
+     * Handle a drop event
+     */
+    function dropHandler(items) {
+      vm.userInput.cfIgnoreFile = false;
+      // Find out what has been dropped and take appropriate action
+      itemDropHelper.identify(items).then(function (info) {
+        vm.userInput.localPath = info.value ? info.value.name : '';
+        if (info.isFiles) {
+          vm.options.wizardCtrl.showBusy('deploy-app-dialog.step-info.scanning');
+          itemDropHelper.traverseFiles(info.value, CF_IGNORE_FILE, CF_DEFAULT_IGNORES).then(function (results) {
+            vm.userInput.fileScanData = results;
+            vm.userInput.sourceType = 'local';
+            vm.options.wizardCtrl.showBusy();
```

Comment:
Added a finally and moved the showBusy into that, so at least the wizard will work again if something fails.